src/root/response.d

```diff
@@ -42,24 +50,19 @@ import ddmd.root.file, ddmd.root.filename;
  *   0   success
  *   !=0   failure (argc, argv unchanged)
  */
-extern (C++) bool response_expand(Strings* args)
+bool response_expand(Strings* args)
 {
-    const(char)* cp;
     int recurse = 0;
-    for (size_t i = 0; i < args.dim;)
+    foreach (i, cp; *args)
```

Comment:
Note line 61. `foreach` can behave erratically if `args` gets resized inside the loop.